README.md

```diff
@@ -25,10 +25,8 @@ The following items are requirements for using Protect with Magento:
 1. `$ sudo bin/magento cache:clean`
 1. `$ sudo bin/magento cache:enable`
 1. `$ sudo bin/magento module:status`
-1. Confirm that NS8 Protect is listed.
+1. After running the commands outlined in steps two through six, you can access NS8 through your Magento instance. 
 
 ## License
 
-See [License](./LICENSE)
-
- © [ns8inc](https://ns8.com)
+For more information about NS8, see our [License](./LICENSE) or visit us at [ns8inc](https://ns8.com).
```

Comment:
In the Dropbox document, the link text at the end of this sentence is `ns8.com` (not `ns8inc`).

Reply:
Good catch; fixed.